<map id="openage::pyinterface::tests::err_cpp_to_py_helper" name="openage::pyinterface::tests::err_cpp_to_py_helper">
<area shape="rect" id="node1" title="Called by cppinterface.demo_cpp_to_py." alt="" coords="5,20,155,76"/>
<area shape="rect" id="node2" href="$namespaceopenage_1_1log.xhtml#a325b0901ddb3f2af36f93781b7dd9744" title="error message" alt="" coords="219,5,329,32"/>
<area shape="rect" id="node3" href="$namespaceopenage_1_1pyinterface_1_1tests.xhtml#a18de7d70653d513f4e6fd5e8cb4fe0cd" title=" " alt="" coords="203,57,345,98"/>
</map>
